world_rot: [-1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 0.0]
tracker2wrist_z_offset: -0.11

mounting:
  left: [0, -1, 0, -1, 0, 0, 0, 0, -1]
  right: [0, -1, 0, -1, 0, 0, 0, 0, -1]

init_tcp:
  left:
    position: [0.749, 0.394, 1.162]
    orientation: [0.03788625, 0.05543418, 0.99774330, -0.99631412, -0.07480106, 0.04198790, 0.07695983, -0.99565649, 0.05239592]
  right:
    position: [0.751, -0.456, 1.167]
    orientation: [-0.09147513, -0.03885954, 0.99504886, 0.99557056, 0.01822147, 0.09223469, -0.02171545, 0.99907853, 0.03702060]

